AudioBar/AudioBarViewController.swift

```diff
@@ -77,8 +76,22 @@ public final class AudioBarViewController: UIViewController, StoreDelegate {
         remoteCommandCenter.skipBackwardCommand.isEnabled = view.isSeekBackButtonEnabled
         remoteCommandCenter.skipForwardCommand.preferredIntervals = [.init(value: view.seekInterval)]
         remoteCommandCenter.skipBackwardCommand.preferredIntervals = [.init(value: view.seekInterval)]
-        nowPlayingInfoCenter.setPlaybackDuration(view.playbackDuration)
-        nowPlayingInfoCenter.setElapsedPlaybackTime(view.elapsedPlaybackTime)
+        var nowPlayingInfo: [String: Any] = [:]
```

Comment:
Sometimes setting each value of `MPNowPlayingInfoCenter.default().nowPlayingInfo` may result in sudden losing some of the values. That's why I'm setting a new dictionary variable, adding all the data here and do: `nowPlayingInfoCenter.nowPlayingInfo = nowPlayingInfo`.